action.yml

```diff
@@ -42,27 +43,27 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - id: phylum-test
-      uses: phylum-dev/install-phylum-latest-action@v1.3
+    - name: Install phylum CLI
+      uses: phylum-dev/install-phylum-latest-action@v1
```

Comment:
Making this change now will allow greater flexibility in updating the `install-phylum-latest-action` GHA going forward.